dashboard/src/app/(authenticated)/(diagram)/components/NodeTypes/Task/TaskDetails.tsx

```diff
@@ -53,9 +53,13 @@ export const TaskDetails: FC<{ task?: TaskNode; nodeRun?: NodeRun }> = ({ task,
           <ol className="list-inside list-decimal">
             {task.variables.map((variable, i) => (
               <li className="mb-1 flex gap-1" key={`variable.${i}`}>
-                <div className="bg-gray-200 px-2 font-mono text-fuchsia-500">arg{i}</div>
+                <div className="bg-gray-200 px-2 font-mono text-fuchsia-500">
+                  {nodeRun ? data?.inputVariables[i].varName : `arg${i}`}
```

Comment:
What if `varName` is null/undefined?

Reply:
Might be better also to use the ternary with `data` since `data` will only be available if `nodeRun` is available

```tsx
{ data ? data... : ... }
```